### PR TITLE
[StorageManager] Add service storage-manager

### PR DESCRIPTION
**Pull request by TongboZhang**
[StorageManager] Add service storage-manager

### CODE REVIEW

File: src/storage-manager/deploy/conf/exports [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# /etc/exports: the access control list for filesystems which may be exported
+#               to NFS clients.  See exports(5).
+#
+# Example for NFSv2 and NFSv3:
+# /srv/homes       hostname1(rw,sync,no_subtree_check) hostname2(ro,sync,no_subtree_check)
+#
+# Example for NFSv4:
+# /srv/nfs4        gss/krb5i(rw,sync,fsid=0,crossmnt,no_subtree_check)
+# /srv/nfs4/homes  gss/krb5i(rw,sync,no_subtree_check)
+/share/pai *(ro,sync,fsid=0,root_squash,no_subtree_check,anonuid=1000,anongid=1000)
```
[the remark this answers is not in the document]
Comment (TongboZhang) [reply src/storage-manager/deploy/conf/exports]: Changed

File: src/storage-manager/build/storage-manager.dockerfile [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      krb5-user \
+      cifs-utils \
+      nfs-common \
+      netbase \
+      nfs-kernel-server \
+      kmod
+
+ENV SHARE_ROOT=/share/pai
+
+COPY deploy/conf/smb.conf /etc/samba/smb.conf
```
[the remark this answers is not in the document]
Comment (TongboZhang) [reply src/storage-manager/build/storage-manager.dockerfile]: Changed

File: src/storage-manager/build/storage-manager.dockerfile [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+FROM ubuntu:16.04
+
+COPY deploy/conf/nsswitch.conf /etc/nsswitch.conf
+COPY deploy/conf/krb5.conf /etc/krb5.conf
```
[the remark this answers is not in the document]
Comment (TongboZhang) [reply src/storage-manager/build/storage-manager.dockerfile]: Changed

**Comment by wangdian** [comment]
Merged into #3826 . Close this PR

**Pull request closed without merging**